bin/GameIndex.yaml

```diff
@@ -4827,6 +4844,8 @@ SCUS-97268:
   compat: 5
   gameFixes:
     - VU0KickstartHack # Fixes Character SPS.
+    - EETimingHack # Fixes DMA errors 
```

Comment:
This timing hack needs to be removed.

Reply:
Have you played through the USversion to make sure it isn't required?